resources/views/auth/subscription_checkout.php

```diff
@@ -144,21 +144,25 @@
         });
 
     <?php elseif (!empty($plan->stripe_id)): ?>
-        $("#zype-card-date").mask("99/99");
-        $("#zype-card-number").mask("9999 9999 9999 9999");
+        $(".zype-card-date").mask("99/99");
```

Comment:
Why shift these to classes instead of ids? Wouldn't IDs be safer?

Reply:
Bc the form is rendered for every video in the page, so we are having more than one input with the same ID

Reply:
Oy vey. Ok, so we'll need to refactor it at some point to have a single form that gets moved around to videos as needed.

Reply:
Yes, we will have to do that. I didn't go with that solution bc it was going to take me longer